frontend.java

```diff
@@ -0,0 +1,540 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
+public class frontend {
+	static int loggedIn = 0;
+	static int type = 0; // if type = 1 its retail, if type=2 its agent
+	static ArrayList<String> transactionFile = new ArrayList<String>();
+	public static void main(String[] args) {
+		login();
```

Comment:
Other than the 'main', why are all methods and data members defined as static?  why not create a constructor and instantiate an object of type Frontend in the "main()"?